websites/S/SuperiorServers.co/dist/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "Nick",
+    "id": "193909360219389952"
+  },
+  "url": ["forum.superiorservers.co", "superiorservers.co"],
+  "description": {
+    "en": "Presence to show off what you are browsing on SuperiorServers.co to your friends."
+  },
```

Comment:
whoops this is wrong actually, this should be a description of the website, not the presence